categories.json

```diff
@@ -27,6 +27,16 @@
 				{"name": "game", "description": "Game servers"},
 				{"name": "web", "description": "Web servers"}
 			]},
+			{"name": "business", "description": "Business applications", "categories": [
+				{"name": "sap", "description": "SAP solutions"},
+				{"name": "salesforce", "description": "Salesforce solutions"},
+				{"name": "workday", "description": "Workday solutions"}
```

Comment:
how about putting all of these 3 categories in a parent category called `"ERP software"` or similar